.github/workflows/sync-cnb.yml

```
@@ -1,16 +1,17 @@
-name: Sync to CNB
-on:
-  push:
-    branches: [main, develop]
+name: sync-cnb
```

```suggestion
name: Sync to CNB
```
个人认为对外展示用的名字还是大写容易辨识。毕竟jobs name已经是连写了